### ISSUE TITLE
Discussion: Add html-loader to base config

### ISSUE THREAD

**Issue by Tibfib** [andywer/webpack-blocks#62]
I think the html-loader is universal enough to add to the base config. 

Thoughts?

https://github.com/andywer/webpack-blocks/blob/master/packages/webpack2/index.js

**Comment by andywer** [comment]
Right, this is a pretty common loader. But it also brings a bunch of additional transitive dependencies (see its [package.json](https://github.com/webpack/html-loader/blob/master/package.json)), thus taking a bit more time to `npm install`.

Guess I will just post a poll on twitter!

**Comment by andywer** [comment]
Here is the poll 😉
https://twitter.com/andywritescode/status/814873242833973248

**Comment by Tibfib** [comment]
Awesome!

I could see it going either way. Since I usually use webpack for angular apps, the html-loader is a no brainer... Not sure if that need extends to react/vuejs development. 

Perhaps it could become a new block? It could be dead simple. Just the html loader.

**Comment by andywer** [comment]
Sure. Let's consider the new block is plan B :)

**Comment by andywer** [comment]
Hmm. The poll suggests creating a block for it, but due to a really bad timing (NYE) participation was extremely bad.

I guess we should nevertheless just create new block. We could also create an Angular2 preset, for example :)
